companion/flashing.md

```diff
@@ -16,23 +16,19 @@ To check power up the transmitter in Bootloader mode:
 ![bootloadersmall](https://user-images.githubusercontent.com/20209851/29190168-1b415e06-7de7-11e7-8d92-0010df929bbd.png)
 
 
-| Tx  | OpenTX Bootloader | PC OS | Add PC Driver |
+| Tx  | OpenTX Bootloader | Flash method |
```

Comment:
I deliberately chose the Add PC driver because users are reading about zadig and installing it... the idea of the table is that from the user point of view do I have to go get and install drivers.... no an under the hood technical document.... the table was posted in issues and discussed... consensus reached and then I posted the changes here.... I neither agree nor disagree with  your changes... it is a different style....

Reply:
I wanted to be precise as the flash method is dfu-util even under windows.